pkgs/build-support/setup-hooks/move-docs.sh

```diff
@@ -6,16 +6,16 @@ preFixupHooks+=(_moveToShare)
 
 _moveToShare() {
     forceShare=${forceShare:=man doc info}
-    if [ -z "$forceShare" -o -z "$out" ]; then return; fi
+    if [ -z "$forceShare" ] || [ -z "$out" ]; then return; fi
```

Comment:
```suggestion
    if [[ -z "$forceShare" || -z "$out" ]]; then return; fi
```
In general, I think we should use the `[[ ]]` bash tests everywhere. They allow that kind of simplified construct, and are also less vulnerable to unescaped variables.

Reply:
I agree, I just followed the shellcheck to the letter here because I wanted this PR to be the least controversial as possible.
Let me change this